mint-auditor/src/db/models/mint_config_tx.rs

```diff
@@ -350,15 +350,18 @@ mod tests {
 
     #[test_with_logger]
     fn insert_enforces_uniqueness(logger: Logger) {
-        let mut rng = mc_util_test_helper::get_seeded_rng();
         let test_db_context = TestDbContext::default();
         let mint_auditor_db = test_db_context.get_db_instance(logger.clone());
         let token_id1 = TokenId::from(1);
+        let token_id2 = TokenId::from(2);
 
         let conn = mint_auditor_db.get_conn().unwrap();
-
+        let mut rng = mc_util_test_helper::get_seeded_rng();
+        let mut rng2 = mc_util_test_helper::get_seeded_rng();
         let (mint_config_tx1, _signers) = create_mint_config_tx_and_signers(token_id1, &mut rng);
         let (mint_config_tx2, _signers) = create_mint_config_tx_and_signers(token_id1, &mut rng);
+        let (mint_config_tx1_tkn2, _signers) =
```

Comment:
Instead of using two rngs, would it be possible to clone the original mint_config_tx_1 and change the token_id in it? I think that is a bit easier to follow

Reply:
Done.